docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -70,7 +70,11 @@ and the user running Oasis Node is `oasis`, you can do:
 sudo adduser oasis sgx
 ```
 
-Failure to do so may result in permission denied errors during runtime startup.
+Then, restart your computer or reload the SGX modules for permissions to take
```

Comment:
Hm, you shouldn't need to restart your computer. However if you are running your node from an interactive session (e.g. a terminal with your user account), then you need to terminate the session (logout) and login again.

Reply:
> Hm, you shouldn't need to restart your computer. However if you are running your node from an interactive session (e.g. a terminal with your user account), then you need to terminate the session (logout) and login again.

Oops, you were right. Closing/Opening the terminal app didn't work, because they're probably forked from the kde session anyway, but logging out of the desktop and back in works. Will fix.